models/_sector/depin/_schema.yml

```diff
@@ -0,0 +1,30 @@
+version: 1
+
+models:
+  - name: depin_revenue
+    meta:
+      blockchain: ['polygon']
+      sector: depin
+      contributors: 0xnirmal
+    config:
+      tags: ['depin', 'revenue']
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - date
+            - chain
+            - name
+            - revenue
```

Comment:
We may need to define the unique_key in the config blocks.
Especially for incremental views.

Reply:
Added in .sql as key, don't think I need to add here given I defined in config block, but lmk otherwise